### DISCUSSION TITLE
Examplenotebook to demonstrate alternative approaches to set up RyRolL calculations

### DISCUSSION THREAD

**Discussion by GRPlan** [pyroll-project/pyroll-core#205] [Show and tell]
To get discussions going on how to use PyRolL I put together and attached an example Jupyter-Notebook with some supporting files.

[v2.zip](https://github.com/pyroll-project/pyroll-core/files/15063518/v2.zip)

You are welcome to check it out and add questions and comments.

**Comment by RollingTechSupport** [comment]
Thank you for the detailed work. We've created a test for `pyroll-basic` using your work.
